tests/e2e/test_load.py

```diff
@@ -0,0 +1,49 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import pytest
+import torch
+
+from azstoragetorch.io import BlobIO
+
+
+@pytest.fixture(scope="module", autouse=True)
+def torch_hub_cache(tmp_path_factory):
+    current_dir = torch.hub.get_dir()
+    yield tmp_path_factory.mktemp("torch_hub")
+    torch.hub.set_dir(current_dir)
+
+
+@pytest.fixture(scope="module")
+def model(torch_hub_cache):
```

Comment:
Now that we are doing `autouse` we should be able to remove reference to the `torch_hub_cache` here.